reader_test.go

```diff
@@ -220,6 +235,16 @@ func TestDecodeMediaPlaylist(t *testing.T) {
 	//fmt.Println(p.Encode().String()), stream.Name}
 }
 
+func TestDecodeMediaPlaylistDoubleHeader(t *testing.T) {
+	assert := assert.New(t)
+	p, err := NewMediaPlaylist(5, 798)
+	if err != nil {
+		t.Fatalf("Create media playlist failed: %s", err)
+	}
+	err = p.Decode(*bytes.NewBuffer([]byte(badMediaPlaylist)), true)
+	assert.Error(err, "Should be error")
```

Comment:
Please make this more descriptive.  For instance, "Multiple headers should return error"

Reply:
Fixed. Please note I've made another small change - made error to be global variable.